docs/faq.rst

```diff
@@ -343,6 +343,19 @@ Please refer to `this great answer from kmike on reddit.
 
 .. _why-lua:
 
+Why are CSS styling and images missing from the .har archive?
```

Comment:
the whole section should be moved above `_why-lua:` label, because it is a label for the next section.

Reply:
yup, moved the section above `_why-lua:` and added its own label